environment.yml

```diff
@@ -1,9 +1,567 @@
 name: farmbeats-sdk-env
-
+channels:
+  - conda-forge
+  - anaconda
+  - defaults
 dependencies:
-  - python=3.6.9
-  - numpy
-  - pandas
-  - rasterio
-  -pip:
-      - azure-farmbeats
\ No newline at end of file
+  - _py-xgboost-mutex=2.0=cpu_0
+  - backports=1.0=py_2
+  - backports.functools_lru_cache=1.6.4=pyhd8ed1ab_0
+  - binutils_impl_linux-64=2.31.1=h6176602_1
+  - binutils_linux-64=2.31.1=h6176602_9
+  - blas=1.0=mkl
+  - bzip2=1.0.8=h7b6447c_0
+  - ca-certificates=2020.12.5=ha878542_0
+  - certifi=2020.12.5=py36h5fab9bb_1
+  - convertdate=2.3.2=pyhd8ed1ab_0
+  - cudatoolkit=10.1.243=h6bb024c_0
+  - cycler=0.10.0=py36_0
```

Comment:
Did we test this on AML compute? Is this yml exported from you local?

Reply:
exported from aml, will be modifying separately on upcoming pr